CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ release`, please list any changes that you have made to the
 document. For an example, see [this
 PR](https://github.com/ARM-software/acle/pull/19).
 
-The branch `latest-release` is pointing at the more recent release of
+The branch `latest-release` is pointing at the most recent release of
```

Comment:
Perhaps it makes sense to merge the two paragraphs on latest-release into one? Seems to me they contain overlapping information and as a reader I get confused why we have two.

Reply:
Done, thanks!